### PR TITLE
a few changes

### PR DESCRIPTION
**Pull request by satra**
@djarecka - a few changes in the introduction to help improve clarity. also i reworked the first class to be more accessible to a new reader.

### CODE REVIEW

**Comment by satra** [comment]
@djarecka - i'll let you merge this.

File: papers/pydra/paper.rst [hunk 1]
```diff
@@ -108,42 +114,54 @@ or for use in applications where caching and distributed execution are not neces
...
+directory. There are several classes that inherit from ``TaskBase`` and each has
+a different application:
 
-* ``FunctionTask`` is a *Task* that is design as a wrapper for Python function.
-  Every Python defined function could be transformed to the ``FunctionTask`` by using Pydra
-  decorator - ``mark.task``.
-  In addition, the user can use Python's function annotation or another Pydra decorator
-  |---| ``mark.annotate`` in order to specify the output, see an example below:
+* ``FunctionTask`` is a *Task* that executes Python functions. Most Python functions
+  declared in an existing library, package, or interactively can be converted to
```
Comment (djarecka) [inline papers/pydra/paper.rst:148]: i'm not sure what you mean in this line

Comment (satra) [reply papers/pydra/paper.rst:148]: it means we can turn existing functions into pydra function tasks?

File: papers/pydra/paper.rst [hunk 2]
```diff
@@ -155,7 +173,11 @@ There are several classes that inherit from ``TaskBase`` and each has a differen
          import statistics as st
          return st.mean(my_data), st.stdev(my_data)
 
-     task = mean_dev(my_data=[...])
+     result = mean_dev(my_data=[...])
```
Comment (djarecka) [inline papers/pydra/paper.rst:176]: this is a task, result you can have when `mean_dev(my_data=[...])()`

File: papers/pydra/paper.rst (hunk 1 above)
Comment (djarecka) [reply papers/pydra/paper.rst:148]: yes, but wasn't sure about this "r interactively", etc., but we can keep it

**Comment by djarecka** [comment]
ok, will merge it and check if i'm able to build pdf

**Pull request merged**